index.html

```diff
@@ -8,62 +8,198 @@
-				<a href="https://github.com/Reggeditt" class="social-link">
-				  <img src="assets/images/githubIcon.png" alt="" class="icon">
-				</a>
-			  </li>
-			  <li>
-				<a href="https://www.linkedin.com/in/tetteh-kodjo-sarso-888989125/" class="social-link">
-			      <img src="assets/images/linkedin.png" alt="" class="icon">
-				</a>
-			  </li>
-			  <li>
-				<a href="#" class="social-link">
-					<img src="assets/images/angelist 1.png" alt="" class="icon">
-				</a> 
-			  </li>
-			  <li>
-				<a href="https://twitter.com/tksarso" class="social-link">
-					<img src="assets/images/twitterIcon.png" alt="" class="icon">
-				</a>
-			  </li>
-			  <li>
-				<a href="#" class="social-link">
-					<img src="assets/images/Vector.png" alt="" class="icon">
-				</a>
-			  </li>
-		  </ul>
+    <section class="hero">
+      <h1 class="header">Hey There. I’m Alex</h1>
+      <span class="sub-header">I’am a Software Developer</span>
+      <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+      <div class="social-icons">
+        <a href="https://github.com/Reggeditt" class="social-link">
+          <img src="assets/images/githubIcon.png" alt="" class="icon">
+        </a>
+        <a href="https://www.linkedin.com/in/tetteh-kodjo-sarso-888989125/" class="social-link">
+          <img src="assets/images/linkedin.png" alt="" class="icon">
+        </a>
+        <a href="#" class="social-link">
+          <img src="assets/images/angelist.png" alt="" class="icon">
+        </a>                
+        <a href="https://twitter.com/tksarso" class="social-link">
+          <img src="assets/images/twitterIcon.png" alt="" class="icon">
+        </a>
+        <a href="#" class="social-link">
+          <img src="assets/images/Vector.png" alt="" class="icon">
+        </a>
+      </div>
+    </section>
+    <section class="works">
+      <div class="works-heading">
+        <h1 class="heading">My Recent Works</h1>
+	  <hr id="works-header-divider">
+      </div>
+      <div class="works-img"></div>
+      <div class="multi-posts">
+            <h2 class="heading">Multi-Post Stories</h2>
+            <p class="sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard 
+				dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+			<div >
+				<ul class="stack-list multi-post-stacks">
+					<li><a href="#">css</a></li>
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
 			</div>
-	</section>
-	 <section class="works"></section>
-	 <section class="about"></section>
-	 <section class="contact"></section>
-  </main>
-  <footer></footer>
+			<button class="multiposts-btn">See Project</button>
+	  </div>
```

Comment:
- [Optional] Kindly, follow the Figma design each by each.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/205356275-5b372fee-395d-42de-8b08-afca0f5b534c.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/205356268-125b41fa-5a81-492a-be0e-91f737e5bda4.png" width="200">|


😎️

🤝️

Reply:
ill get on it! thanks!

Reply:
fixed it